src/software/thunderscope/replay/proto_player.py

```diff
@@ -119,7 +183,9 @@ def load_replay_chunk(replay_chunk_path: str) -> list:
                     line = log_file.readline()
                     if not line:
                         break
-                    cached_data.append(line)
+
+                    if not ProtoPlayer.is_log_entry_corrupt(line):
+                        cached_data.append(line)
                 except EOFError:
```

Comment:
this try-except could be more robust

`readline()` could fail if a replay file is corrupt

Reply:
I've added a new line as shown below.